common/stream-model.cpp

```diff
@@ -779,177 +773,209 @@ namespace rs2
-                                ImColor(alpha(sensor_bg, 0.1f)));
+                line_y += ImGui::GetTextLineHeight() + 3;
+            }
+            else
+            {
+                std::string text = "";
+                if( at.name != "" )
+                    text = rsutils::string::from() << at.name << ":";
 
-                        ImGui::PushStyleColor(ImGuiCol_Text, redish);
-                        ImGui::Text("%s", text);
-                        ImGui::PopStyleColor();
+                ImGui::SetCursorPosY( line_y );
+                ImGui::PushStyleColor( ImGuiCol_Text, white );
+                ImGui::Text( "%s", text.c_str() );
+                ImGui::SameLine();
 
-                        line_y += ImGui::GetTextLineHeight() + 3;
-                    }
-                    else
+                if( at.description != "" )
+                {
+                    if( ImGui::IsItemHovered() )
                     {
-                        std::string text = "";
-                        if (at.name != "") text = rsutils::string::from() << at.name << ":";
-                        auto size = ImGui::CalcTextSize(text.c_str());
-
-                        for (int i = 3; i > 0; i -= 1)
-                            ImGui::GetWindowDrawList()->AddRectFilled({ curr_info_rect.x + 10 - i, line_y - i },
-                                { curr_info_rect.x + 10 + i + size.x, line_y + size.y + i },
-                                ImColor(alpha(sensor_bg, 0.1f)));
-
-                        ImGui::PushStyleColor(ImGuiCol_Text, white);
-                        ImGui::Text("%s", text.c_str()); ImGui::SameLine();
-
-                        if (at.description != "")
-                        {
-                            if (ImGui::IsItemHovered())
-                            {
-                                ImGui::SetTooltip("%s", at.description.c_str());
-                            }
-                        }
-
-                        text = at.value;
-                        size = ImGui::CalcTextSize(text.c_str());
-
-                        for (int i = 3; i > 0; i -= 1)
-                            ImGui::GetWindowDrawList()->AddRectFilled({ curr_info_rect.x + 20 + max_text_width - i, line_y - i },
-                                { curr_info_rect.x + 30 + max_text_width + i + size.x, line_y + size.y + i },
-                                ImColor(alpha(sensor_bg, 0.1f)));
-
-                        ImGui::PopStyleColor();
-
-                        ImGui::SetCursorScreenPos({ curr_info_rect.x + 20 + max_text_width, line_y });
+                        ImGui::SetTooltip( "%s", at.description.c_str() );
+                    }
+                }
 
-                        std::string id = rsutils::string::from() << "##" << at.name << "-" << profile.unique_id();
+                text = at.value;
+                
+                ImGui::SetCursorPosX( 20 + max_text_width );
```

Comment:
Is that the offset to the values?
so we can use 20 instead of * 2 in the background calculation? (maybe with some more offset)?

Reply:
20 is space between the first and second columns.

Reply:
I think 20 is the space between the end of the first column and the start of the second one meaning:

1                               2    3
---------------------------------
Frame Counter               100 
Some Long Metadata     5

2 = longest metadata type name
3 = Start of value column

right?
3-2=20

Reply:
Yes, 20 is space between the end of the first column ( end of longest metadata type name) and the start position of the value column.